.github/workflows/pull_request.yaml

```diff
@@ -9,7 +9,6 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
-    container: "python:3.10-slim"
```

Comment:
dont drop this line

Reply:
This line is the one in conflict with 

```yml
- uses: actions/setup-python@v4
        with:
          python-version-file: .python-version
          cache: 'poetry'
```

Just for testing right now